### ISSUE TITLE
Unable to Move/Rotate Starfield BSGeometry Nodes

### ISSUE THREAD

**Issue by eiasjsf** [hexabits/nifskope#57]
I'm not sure if I'm doing something wrong or it just hasn't been implemented yet, but I'm unable to move or rotate Starfield BSGeometry nodes while scale seems to work.

**Comment by eiasjsf** [comment]
I think I figured it out, I was trying to edit it using the right click -> transform method which didn't work but changing the translation/rotation in the block details seems to have an effect.

Edit: Apparently it doesn't actually work, I can adjust the scale/position/rotation but when in-game the changes aren't applied.

**Comment by hexabits** [comment]
@eiasjsf BSGeometry isn't a node.  Try parenting it to an NiNode and moving that.

**Comment by hexabits** [comment]
Also, just to confirm, you know that loose files work for you otherwise?

**Comment by eiasjsf** [comment]
Yep, loose files work. I'm still trying to figure this out, but I think it comes down to user error on my part. It's been several years since I messed with altering nifs directly in Nifskope instead of Outfit Studio or something, so I guess I'd forgotten more than I realized. Thanks for responding and your continuing work on Nifskope.
